src/components/molecules/PlanRoom/Tittle/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { Box } from '@chakra-ui/react';
+
+const PlanRoomTitle = () => (
```

Comment:
The folder and component names should be the same. In addition, you've got a typo there.

Reply:
To be corrected